{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -81,7 +81,8 @@ Changelog = "{{ cookiecutter.url }}/releases"
 Chen2020 = "{{ cookiecutter.__project_slug }}.parameters.input.Chen2020:get_parameter_values"
 
 [project.entry-points."models"]
-SPM = "{{ cookiecutter.__project_slug }}.models.input.SPM:SPM"
+SPM = "{{ cookiecutter.__project_slug }}.models.input.spm:SPM"
+Reservoir = "{{ cookiecutter.__project_slug }}.models.input.basic_reservoir:BasicReservoir"
```

Comment:
```suggestion
BasicReservoir = "{{ cookiecutter.__project_slug }}.models.input.basic_reservoir:BasicReservoir"
```

Reply:
Just merged your suggestions hoping that it'd fix the failures

Reply:
To fix the failures we would have to change the model names to the same ones as their keys in entry points. I will do it real quick!

Reply:
It should work now, the assertion was failing because there was a list comparison and the lists weren't sorted so the unordered elements were causing the assertion to fail.